extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -167,9 +167,9 @@ public function getParseResponseFromPageId( $pageId ) {
 					'action'	=> 'parse',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "action" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)